react/src/pages/UsersPage.js

```diff
@@ -174,15 +174,15 @@ const UsersPage = () => {
                                 <span id={`tooltip-users-status-${user.status}`}>{userMetadata.statusTitle(user.status) || Char.EmptySet}</span>
                                 <Tooltip id={`tooltip-users-status-${user.status}`} position="bottom" size="small" text={`Status: ${user.status}`} />
                             </td>
-                            <td style={tdStyle}>
+                            <td style={{...tdStyle,whiteSpace:"nowrap"}}>
```

Comment:
I'm not positive I am using the right syntax here, but is there some reason you don't do 
```
const tdStyle = { verticalAlign: "top", paddingRight: "6pt", paddingTop: "4pt", paddingBottom: "8pt" };
const tdStyleNoWrap = {...tdStyle,whiteSpace:"nowrap"};
```
above and then later use (untested):
```
<td style={{tdStyleNoWrap}}>
```
?

Reply:
Yep, good call, thanks.